pkgs/development/compilers/lessc/default.nix

```diff
@@ -23,7 +23,7 @@ stdenv.mkDerivation rec {
     description = "LESS to CSS compiler";
     homepage = http://lesscss.org/;
     license = licenses.asl20;
-    platforms = platforms.linux;
+    platforms = platforms.linux ++ platforms.darwin;
```

Comment:
I don't really know much about systems, my friend told me adding darwin would make it work on my machine. The less docs don't really specify what systems it runs on either. So the only reason is that I haven't tried it and don't know...